install_magenta.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Check which operating system
+if [ "$(uname)" == "Darwin" ]; then
+    echo "Mac OS Detected"
+    export OS="MAC"
```

Comment:
since these don't need to be exported to the environment, 'readonly' might be a better choice.

https://google.github.io/styleguide/shell.xml?showone=Constants_and_Environment_Variable_Names#Constants_and_Environment_Variable_Names